common/rpc/interceptor/telemetry.go

```diff
@@ -146,6 +146,11 @@ func (ti *TelemetryInterceptor) unaryOverrideOperationTag(fullName, operation st
 			if request.GetWaitNewEvent() {
 				return metrics.FrontendPollWorkflowExecutionHistoryScope
 			}
+		} else if operation == metrics.HistoryGetWorkflowExecutionHistoryScope {
+			request := req.(*workflowservice.GetWorkflowExecutionHistoryRequest)
```

Comment:
I think we need a separate condition for history service requests. Not positive but I don't think they will have the `api.WorkflowServicePrefix`, I think instead they will have the string defined by `service/history/handler#serviceName` constant.

I do know for sure the request type will not be `*workflowservice.GetWorkflowExecutionHistoryRequest`, it will be `*historyservice.GetWorkflowExecutionHistoryRequest` (which contains the original workflowservice request, if relevant)